com.unity.robotics.urdf-importer/Runtime/Unity.Robotics.URDFImporter.asmdef

```diff
@@ -2,7 +2,8 @@
     "name": "Unity.Robotics.UrdfImporter",
     "rootNamespace": "Unity.Robotics.UrdfImporter.Urdf",
     "references": [
-        "GUID:6a697808d7c80a549b57420070d6c4f3"
+        "GUID:6a697808d7c80a549b57420070d6c4f3",
+        "GUID:7c3cd0727030049da8a847e9ce61c46c"
```

Comment:
Deactivate the "Use GUIDs" in the assembly definition. This is so that it is easier to track in source control and also to find out when a dependency is missing.